spec/models/participant_profile/npq_spec.rb

```diff
@@ -1,101 +0,0 @@
-# frozen_string_literal: true
-
-require "rails_helper"
-
-RSpec.describe ParticipantProfile::NPQ, type: :model do
-  let(:npq_application) { create(:npq_application, :accepted) }
-  let(:profile) { npq_application.profile }
-
-  describe "#withdrawn_for" do
```

Comment:
since we are deleting the specs for the model, maybe we can delete the methods in the model as well but keep the skeleton model itself to avoid them living a bit longer without specs 🤔